src/pages/saksbehandling/søknadsbehandling/formue/Formue.tsx

```diff
@@ -109,7 +109,7 @@ const schema = yup
             .typeError('Feltet må fylles ut'),
         epsFnr: yup.mixed<string>().when('borSøkerMedEPS', {
             is: true,
-            then: yup.string().required('Du må legge inn ektefelle/samboers fødselsnummer').length(11),
+            then: yup.string().typeError('Du må legge inn ektefelle/samboers fødselsnummer').length(11),
```

Comment:
Den er fortsatt spnn at man må fylle inn?

Reply:
Hva er det dette egentlig betyr? At hvis det blir en type-error på fnr-strengen så får man valideringsfeilmeldingen (og så må den være length 11 også)?

Reply:
@simejo Yes man må fylle in den hvis `borSøkerMedEps` er true.

@christianmosveen 
Ja, stemmer. Tidligere så fick vi en väldigt teknisk feilmelding fordi hvis bruker ikke fyllde in något så var det null og ikke en string (den feilet da på `yup.string()`-sjekken). Nå sjekker vi att det er en **type av string** i og att lengden er 11.